analysis/tests/common.py

```diff
@@ -0,0 +1,73 @@
+# Copyright 2022 OpenMined.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""Common functions for utility analysis tests"""
+
+import dataclasses
+import enum
+import typing
+
+from absl.testing import parameterized
+
+
+def _assert_dataclasses_are_equal(test: parameterized.TestCase,
```

Comment:
Nit: remove _ in the name (it's not private function in this file anymore)

Reply:
Good point, thanks!